politeiawww/comments.go

```diff
@@ -131,29 +139,50 @@ func (p *politeiawww) updateCommentScore(token, commentID string) (int64, error)
 		case prevAction == 0:
 			// No previous action so we add the new action to the
 			// vote score
-			score += action
+			switch action {
```

Comment:
One last thing.  The -1 and +1 should be defined as global consts so that this becomes something like:

```
			switch action {
			case actionDownvote:
				votes.down += 1
			case actionUpvote:
				votes.up += 1
			}
```

Reply:
Done.

Reply:
Can you move the definitions to the top of the file and add a comment explaining what they are?

Reply:
Can't they also be strings now?  The api defines the `LikeComment.Action` as a string of either "-1" or "1".  We previously converted them to ints because we were adding them directly to the comment score.  Since we're using `counter` to track the upvotes and downvotes now, seems like we can keep `actionDownvote` and `actionUpvote` and strings and compare them directly, right?

Reply:
Good catch! Should've seen that myself :-P

Reply:
Made the changes requested.